app/api/business/brief/brief_business.py

```diff
@@ -13,6 +16,25 @@ def is_open_to_all(brief):
     return False
 
 
+def can_submit_response_to_brief(brief, user, check_response_limit=True):
+    user_status = BriefUserStatus(brief, user)
+
+    if user_status.has_supplier_errors() or not user_status.is_approved_seller():
+        return False, 'Supplier is invalid'
+
+    if brief.lot.slug != 'atm' or brief.data.get('openTo', '') == 'category':
+        if not user_status.is_assessed_for_category():
+            return False, 'Supplier is not assessed for the category of the opportunity'
+
+    if not user_status.is_invited():
```

Comment:
What happens if the brief slug is `atm`?  Will this function always return true?

Reply:
`user_status.is_invited()` doesn't actually check brief lots at all, it checks the `openTo`, `sellerSelector` and invited sellers list. For ATM, it returns true if `openTo` is `'all'`, or if `openTo` is `'category'` and they are assessed for the brief's category.